monitor/consul/watcher.go

```diff
@@ -69,12 +88,18 @@ func (w *Watcher) Watch(ctx context.Context, ch chan<- []*change.Change, chErr c
 		}
 		w.pp = append(w.pp, pl)
 		go func(tp, key string) {
-			err := pl.Run(w.addr)
+			//TODO: Logger...
+			output := pl.LogOutput
+			if output == nil {
+				output = os.Stderr
+			}
+			logger := log.New(output, "", log.LstdFlags)
+			err := pl.RunWithClientAndLogger(w.cl, logger)
 			if err != nil {
 				if chErr != nil {
 					chErr <- err
 				}
-				log.Errorf("plan %s of type %s failed: %v", tp, key, err)
+				harvesterlog.Errorf("plan %s of type %s failed: %v", tp, key, err)
```

Comment:
Instead of using two logging packages:
`"log"`
`"github.com/taxibeat/harvester/log"`

We could encapsulate the logger functionality inside the harvesterlog. 
I believe you have something like that in mind since I saw a TODO tag on `line 91`, is that right?

Reply:
yes the TODO is there to handle the loggers.
harvester should not have it's own log implementation but should be able to support any logger provided of use std log if nothing is provided.
check log package...